dynamodb/src/main/scala/zio/dynamodb/DynamoDBQuery.scala

```diff
@@ -551,27 +552,37 @@ object DynamoDBQuery {
     )
   }
 
+  final case class BatchWriteItemResponse(
```

Comment:
could we put this in the companion object for `BatchWriteItem` and call it just `Response` like we do for `BatchGetItem.Response` ?